datasource/netcdf_groundwater.py

```diff
@@ -75,48 +152,120 @@ def line_type_of(self, line_idx):
     def timestamps(self):
         return self.get_timestamps()
 
+    # def get_timeseries(
+    #         self, object_type, object_id, nc_variable, fill_value=None):
+    #     return self.temporary_get_timeseries_impl(
+    #         object_type, object_id, nc_variable, fill_value)
+
+    def get_timeseries_schematisation_layer(
```

Comment:
Probably should rename to `_get_timeseries_schematisation_layer` to indicate it's private